.vscode/aiven.code-snippets

```diff
@@ -13,6 +13,17 @@
     "description": "Inserts a related pages section"
   },
 
+  "Required access": {
+    "scope": "markdown",
+    "prefix": "Required access",
+    "body": [
+      "## Required access",
```

Comment:
Nice to see you're looking into snippets 👌  

As far as the content goes, we should do our best not to duplicate titles, so we don't end up with many identical H2 by design and negatively impact SEO. Maybe this should be an admonition?

Reply:
I also realize we certainly did this for the related links bit but I'll modify it to not make it a _real_ h2.